### PR TITLE
Update test.ymls to use generated secrets

### PR DESCRIPTION
**Pull request by alzimmermsft**
# Description

Updates `ci.test.yml`'s to use the secret generated during resource deployment.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by benbp — APPROVED** [review]

**Comment by alzimmermsft** [comment]
Live tests for Monitor libraries, Personalizer, and SchemaRegistry will be ignored as they were failing before.

**Review by conniey — APPROVED** [review]

File: sdk/appconfiguration/tests.native.yml [hunk 1]
```diff
@@ -23,9 +23,8 @@ stages:
       TimeoutInMinutes: 90
```
Comment (weshaggard) [inline sdk/appconfiguration/tests.native.yml:23]: Do you know that context of tests.native? What is the purpose of this over the other test pipelines?

Comment (alzimmermsft) [reply sdk/appconfiguration/tests.native.yml:23]: This performs a GraalVM build, @srnagar would know more

**Pull request merged**